iOSDeviceManager_test

```diff
@@ -0,0 +1,13 @@
+bundle install
```

Comment:
Is it supposed to be here?

Reply:
I created this file as a hint. I think it could be removed.